packages/sui-react-web-vitals/src/index.js

```diff
@@ -91,51 +80,82 @@ export default function WebVitalsReporter({
       }
     }
 
+    const computeINPMetrics = entry => {
```

Comment:
```suggestion
    // RenderTime is an estimate because duration is rounded and may get rounded down.
    // In rare cases, it can be less than processingEnd and that breaks performance.measure().
    // Let's ensure it's at least 4ms in those cases so you can barely see it.
    const computeINPMetrics = entry => {
```

Reply:
👍🏻  I just added it